MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -400,13 +399,26 @@ - (void)checkVideoData {
     if ([currentPlayerAsset isKindOfClass:AVURLAsset.class]) {
         AVURLAsset *urlAsset = (AVURLAsset *)currentPlayerAsset;
         NSString * urlString = [[urlAsset URL] absoluteString];
-        if (!_videoURL || [_videoURL isEqualToString:urlString]) {
+        if (!_videoURL || ![_videoURL isEqualToString:urlString]) {
             _videoURL = urlString;
             videoDataUpdated = YES;
         }
     }
-    if (_lastAdvertisedBitrate > 0) {
+    if (![@(_lastDispatchedAdvertisedBitrate) doubleValueIsEqual:@(_lastAdvertisedBitrate)]) {
```

Comment:
This is the magic, set `_sourceDimensionsHaveChanged` because we know the advertised bitrate has changed